packages/vike-vue/src/renderer/createVueApp.ts

```diff
@@ -10,22 +10,25 @@ import { isPlainObject } from '../utils/isPlainObject'
 import { setData } from '../hooks/useData'
 
 type ChangePage = (pageContext: PageContext) => Promise<void>
-async function createVueApp(pageContext: PageContext, ssr: boolean, rootComponentName: 'Head' | 'Page') {
-  const rootComponentRef = ref(markRaw(pageContext.config[rootComponentName]))
-  const layoutRef = ref(markRaw(pageContext.config.Layout))
+async function createVueApp(pageContext: PageContext, ssr: boolean, mainComponentName: 'Head' | 'Page') {
+  const mainComponentRef = ref(markRaw(pageContext.config[mainComponentName]))
```

Comment:
These 2 can be `shallowRef` as far as I'm concerned, but I think `ref(markRaw(x))` does the same thing - I'm not very sure, I have to check